doc/api/prev_api_changes/api_changes_0.98.0.rst

```diff
@@ -84,84 +84,95 @@ with a verb in the present tense.
------------------------------------------------------------- ------------------------------------------------------------
-`identity_transform()`                                       :class:`matplotlib.transforms.IdentityTransform`
------------------------------------------------------------- ------------------------------------------------------------
-`blend_xy_sep_transform(xtrans, ytrans)`                     `blended_transform_factory(xtrans, ytrans)`
------------------------------------------------------------- ------------------------------------------------------------
-`scale_transform(xs, ys)`                                    `Affine2D().scale(xs[, ys])`
------------------------------------------------------------- ------------------------------------------------------------
-`get_bbox_transform(boxin, boxout)`                          `BboxTransform(boxin, boxout)` or
-                                                             `BboxTransformFrom(boxin)` or
-                                                             `BboxTransformTo(boxout)`
------------------------------------------------------------- ------------------------------------------------------------
-`Transform.seq_xy_tup(points)`                               `Transform.transform(points)`
------------------------------------------------------------- ------------------------------------------------------------
-`Transform.inverse_xy_tup(points)`                           `Transform.inverted().transform(points)`
-============================================================ ============================================================
++--------------------------------------------+------------------------------------------------------+
+| Old method                                 | New method                                           |
++============================================+======================================================+
+| ``Bbox.get_bounds``                        | :attr:`.transforms.Bbox.bounds`                      |
++--------------------------------------------+------------------------------------------------------+
+| ``Bbox.width``                             | :attr:`transforms.Bbox.width                         |
+|                                            | <.transforms.BboxBase.width>`                        |
++--------------------------------------------+------------------------------------------------------+
+| ``Bbox.height``                            | :attr:`transforms.Bbox.height                        |
+|                                            | <.transforms.BboxBase.height>`                       |
++--------------------------------------------+------------------------------------------------------+
+| ``Bbox.intervalx().get_bounds()``          | :attr:`.transforms.Bbox.intervalx`                   |
+| ``Bbox.intervalx().set_bounds()``          | [It is now a property.]                              |
```

Comment:
Needed to change to a grid table because the first column cannot contain multi-line cells.